"open last exception in ide" leads to exception

### Describe the bug

Create a sample quarkus application
mvn quarkus:dev
Press x
-> exception from below happens

Apperantly quarkus is searrching for eclipse, but can not find it. Thus printing an exception.
I have no eclipse installed atm, only IDEA.

### Expected behavior

_No response_

### Actual behavior

2025-08-01 14:07:25,008 ERROR [io.qua.dep.con.ConsoleProcessor] (Launch in IDE Action) Failed to open IDE: io.smallrye.common.process.PipelineExecutionException: Failed to start process pipeline                                                                                                                    
        at io.smallrye.common.process.ProcessRunner.initialize(ProcessRunner.java:127)
        at io.smallrye.common.process.ProcessRunner.run(ProcessRunner.java:88)
        at io.smallrye.common.process.ProcessBuilderImpl.run(ProcessBuilderImpl.java:199)
        at io.smallrye.common.process.ProcessBuilder.exec(ProcessBuilder.java:211)
        at io.smallrye.common.process.ProcessBuilder.exec(ProcessBuilder.java:231)
        at io.quarkus.deployment.console.ConsoleProcessor$8.run(ConsoleProcessor.java:188)
        at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: java.io.IOException: Cannot run program "eclipse": CreateProcess error=2, Das System kann die angegebene Datei nicht finden
        at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1170)
        at java.base/java.lang.ProcessBuilder.startPipeline(ProcessBuilder.java:1331)
        at io.smallrye.common.process.PipelineRunner.startProcesses(PipelineRunner.java:618)
        at io.smallrye.common.process.ProcessRunner.initialize(ProcessRunner.java:119)
        ... 6 more
Caused by: java.io.IOException: CreateProcess error=2, Das System kann die angegebene Datei nicht finden
        at java.base/java.lang.ProcessImpl.create(Native Method)
        at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:506)
        at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:159)
        at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1126)
        ... 9 more


### How to Reproduce?

_No response_

### Output of `uname -a` or `ver`

MINGW64_NT-10.0-26100 NANBCHL9NG3 3.3.6-341.x86_64 2022-09-05 20:28 UTC x86_64 Msys

### Output of `java -version`

openjdk 21.0.7 2025-04-15 LTS OpenJDK Runtime Environment Temurin-21.0.7+6 (build 21.0.7+6-LTS) OpenJDK 64-Bit Server VM Temurin-21.0.7+6 (build 21.0.7+6-LTS, mixed mode, sharing)

### Quarkus version or git rev

3.25.0

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

Apache Maven 3.9.11 (3e54c93a704957b63ee3494413a2b544fd3d825b) Maven home: C:\eclipse\tools\java\maven Java version: 21.0.7, vendor: Eclipse Adoptium, runtime: C:\eclipse\tools\java\21 Default locale: de_DE, platform encoding: UTF-8 OS name: "windows 11", version: "10.0", arch: "amd64", family: "windows"

### Additional information

_No response_

--- a/core/deployment/src/main/java/io/quarkus/deployment/ide/IdeProcessor.java
+++ b/core/deployment/src/main/java/io/quarkus/deployment/ide/IdeProcessor.java
@@ -38,7 +38,8 @@ public class IdeProcessor {
 
         IDE_PROCESSES.put(
                 (processInfo -> (processInfo.containInCommand("idea") || processInfo.containInCommand("IDEA"))
-                        && (processInfo.command.endsWith("java") || processInfo.command.endsWith("java.exe"))),
+                        && (processInfo.command.endsWith("java") || processInfo.command.endsWith("java.exe")
+                                || processInfo.command.endsWith("idea64.exe") || processInfo.command.endsWith("idea64"))),
                 Ide.IDEA);
         IDE_PROCESSES.put((processInfo -> processInfo.containInCommand("code")), Ide.VSCODE);
         IDE_PROCESSES.put((processInfo -> processInfo.containInCommand("eclipse")), Ide.ECLIPSE);

